acceptance/tests/reports/submission.rb

```diff
@@ -71,7 +71,8 @@ def query_last_report_time_on(agent)
     agents.each do |agent|
       on(agent, puppet('agent', "-t --server #{master}"))
 
-      on master, "grep -q #{agent} #{testdir}/*/*"
+      hostname = fact_on(agent, 'hostname')
```

Comment:
@joshcooper you are correct. In this context we are looking to verify `agent.node_name` which uses `puppet agent --configprint node_name_value`. I will update accordingly.